### ISSUE TITLE
[a11y] Mega menu is not part of nav landmark

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1654]
## Currently
Most Gen 2 page elements are contained inside the appropriate [HTML content section elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#content_sectioning), however the mega menu and skip to content links are sitting loose on their own, with no content section wrapping them.

![Screen Shot 2021-06-04 at 10.42.41 am.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/72643f59-6391-4cc4-968e-ee2804b6babf)

## Proposed change

- [x] Mega menu should be wrapped along with breadcrumbs in a [`nav` content section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)
- [x] Ideally, SVG sprites should be removed from this area and placed at the bottom of the `body`

**Comment by gepifanio** [comment]
Ideally, SVG sprites should be removed from this area and placed at the bottom of the body - pr: https://github.com/unimelb/pattern-lib/pull/1656

**Comment by gepifanio** [comment]
https://github.com/unimelb/pattern-lib/pull/1705
